README.md

```diff
@@ -228,6 +228,8 @@ public class CustomTokenEnhancer implements TokenEnhancer {
 Now, add it to the Authorization Server configuration:
  
 ```java
+@Configuration
+@EnableAuthorizationServer
```

Comment:
This is the wrong annotation (it's also deprecated)

Reply:
removed